pom.xml

```diff
@@ -247,6 +247,8 @@
               <goal>shade</goal>
             </goals>
             <configuration>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>deploy</shadedClassifierName>
```

Comment:
I renamed to `all-deps`, 'deploy' jars are a bazel-ism: http://bazel.io/docs/be/java.html#java_binary_implicit_outputs